dvc/command/config.py

```diff
@@ -67,6 +67,11 @@ def _list(self):
             return 1
 
         levels = [self.args.level] if self.args.level else self.config.LEVELS
+        if self.config.dvc_dir is None:
+            levels = [
+                level for level in levels if level not in ("repo", "local")
+            ]
+
```

Comment:
It looks like this issue also affects reading config variables outside of a DVC repo (`dvc config <name>), it's not specific to `--list`.

It should probably be fixed in both `_list()` and `_get()`